theme/blockly-core.less

```diff
@@ -373,3 +393,45 @@ text.blocklyCheckbox {
         font-size: 17pt !important;
     }
 }
+
+
+/*******************************
```

Comment:
We should review this CSS for unexpected impact when the experiment is not enabled.